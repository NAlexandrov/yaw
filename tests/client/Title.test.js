import React from 'react';
import Title from '../../source/client/components/Title';

describe('<Title />', () => {
  test('should render a title', () => {
    const wrapper = shallow(<Title />);
    expect(wrapper).toMatchSnapshot();
  });
});
