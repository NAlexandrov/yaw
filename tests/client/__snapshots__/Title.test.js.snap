// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should render a title 1`] = `<Styled(h2) />`;
